S/SPRAL/build_tarballs.jl

```diff
@@ -68,18 +69,19 @@ platforms = supported_platforms()
 # The products that we will ensure are always built
 products = [
```

Comment:
This argument must be `Vector{Product}`
```suggestion
products = Product[
```